models/room.py

```diff
@@ -0,0 +1,19 @@
+
+class Room(object):
+    """Room is the parent and Office and Living_space inherit from it"""
+
+    def __init__(self):
+        """both Office and Living_space have access to name"""
+        pass
+
+
+class Office(Room):
+    """office model has capacity of 6"""
+    room_type = 'O'
+    capacity = 6
+
+
+class Living_space(Room):
```

Comment:
**Code Quality: Critical**

Use `PascalCase` for class names. This should be `LivingSpace` and not `Living_space`. See PEP-8.